### PR TITLE
Make manifest section of Bundled HTTP Exchanges optional

### PR DESCRIPTION
**Pull request by horo-t**
Currently there is no need to make "manifest" section mandatory.

Chromium side CL: https://crrev.com/c/1861614

### CODE REVIEW

**Comment by irori** [comment]
(This fixes #499)

**Comment by irori** [comment]
[Abstract](https://wicg.github.io/webpackage/draft-yasskin-wpack-bundled-exchanges.html#rfc.abstract), [section 2](https://wicg.github.io/webpackage/draft-yasskin-wpack-bundled-exchanges.html#rfc.section.2) and [section 2.2](https://wicg.github.io/webpackage/draft-yasskin-wpack-bundled-exchanges.html#rfc.section.2.2) are also mentioning about manifest as mandatory resource. Would you change those as well?

**Comment by horo-t** [comment]
> [Abstract](https://wicg.github.io/webpackage/draft-yasskin-wpack-bundled-exchanges.html#rfc.abstract), [section 2](https://wicg.github.io/webpackage/draft-yasskin-wpack-bundled-exchanges.html#rfc.section.2) and [section 2.2](https://wicg.github.io/webpackage/draft-yasskin-wpack-bundled-exchanges.html#rfc.section.2.2) are also mentioning about manifest as mandatory resource. Would you change those as well?

Done

**Comment by horo-t** [comment]
Marked as non substantive for IPR from ash-nazg.

File: draft-yasskin-wpack-bundled-exchanges.md [hunk 1]
```diff
@@ -92,8 +93,8 @@ Standard ({{INFRA}}).
 
 # Semantics {#semantics}
 
-A bundle is logically a set of HTTP exchanges, with a URL identifying the
-manifest(s) of the bundle itself.
+A bundle is logically a set of HTTP exchanges, with a primary URL in the bundle
+and an optional URL identifying the manifest(s) of the bundle itself.
```
Comment (irori) [inline draft-yasskin-wpack-bundled-exchanges.md]: No need to mention manifest here. How about this?
```
A bundle is logically a set of HTTP exchanges, with a URL identifying the
primary resource of the bundle.
```

File: draft-yasskin-wpack-bundled-exchanges.md [hunk 2]
```diff
@@ -134,7 +135,7 @@ metadata, and if one matches, load that request's response.
 This takes the bundle's stream and returns either an error (where an error is a
 "format error" or a "version error"), an error with a fallback URL (which is
 also the primaryUrl when the bundle parses successfully), or a map ({{INFRA}})
-of metadata containing at least keys named:
+of metadata containing keys named:
```
Comment (irori) [inline draft-yasskin-wpack-bundled-exchanges.md]: I think you can just remove the `manifest` item below (L160-164) and leave the other text in this section unchanged. Section 3.3.2 has all the information about the manifest section.

File: draft-yasskin-wpack-bundled-exchanges.md [hunk 3]
```diff
@@ -369,8 +371,8 @@ steps, taking the `stream` as input.
 
 1. Assert: `metadata` has an entry with the key "primaryUrl".
 
-1. If `metadata` doesn't have entries with keys "requests" and "manifest",
-   return a "format error" with `fallbackUrl`.
+1. If `metadata` doesn't have entries with keys "requests", return a
```
Comment (irori) [inline draft-yasskin-wpack-bundled-exchanges.md]: s/keys/key/

Comment (horo-t) [reply draft-yasskin-wpack-bundled-exchanges.md]: done

File: draft-yasskin-wpack-bundled-exchanges.md (hunk 2 above)
Comment (horo-t) [reply draft-yasskin-wpack-bundled-exchanges.md]: done

File: draft-yasskin-wpack-bundled-exchanges.md (hunk 1 above)
Comment (horo-t) [reply draft-yasskin-wpack-bundled-exchanges.md]: done

File: draft-yasskin-wpack-bundled-exchanges.md [hunk 4]
```diff
@@ -369,8 +364,8 @@ steps, taking the `stream` as input.
 
 1. Assert: `metadata` has an entry with the key "primaryUrl".
 
-1. If `metadata` doesn't have entries with keys "requests" and "manifest",
-   return a "format error" with `fallbackUrl`.
+1. If `metadata` doesn't have entries with key "requests", return a
```
Comment (irori) [inline draft-yasskin-wpack-bundled-exchanges.md]: s/entries/an entry/
(sorry for not noticing in the previous round)

Comment (horo-t) [reply draft-yasskin-wpack-bundled-exchanges.md]: done

**Pull request merged**